examples/wc-wallet-react/src/index.tsx

```diff
@@ -9,17 +9,26 @@ import {
   DEFAULT_METHODS,
 } from './constants'
 import App from './App'
-import { WalletConnectWalletProvider } from '@cityofzion/wallet-connect-sdk-wallet-react'
+import {
+  DEFAULT_AUTO_ACCEPT_METHODS,
+  TOptions,
+  WalletConnectWalletProvider,
+} from '@cityofzion/wallet-connect-sdk-wallet-react'
 import { AccountContextProvider } from './context/AccountContext'
 
-const wcOptions = {
+const wcOptions: TOptions = {
   clientOptions: {
     projectId: DEFAULT_PROJECT_ID,
     metadata: DEFAULT_APP_METADATA,
     logger: DEFAULT_LOGGER,
     relayUrl: DEFAULT_RELAY_PROVIDER,
   },
-  methods: DEFAULT_METHODS,
+  methods: {
+    neo3: DEFAULT_METHODS,
+  },
+  autoAcceptMethods: {
+    neo3: DEFAULT_AUTO_ACCEPT_METHODS,
+  },
```

Comment:
This is outdated, right? You changed this to `blockchains`. Be sure you tested the newest version.